### DISCUSSION TITLE
Guidance / examples for VRP with driver scheduling?

### DISCUSSION THREAD

**Discussion by rust-jvm-go** [TimefoldAI/timefold-solver#442] [General]
My apologies, this is a re-post of: https://stackoverflow.com/questions/77534642/is-this-a-type-of-multi-stage-planning-problem

I'm new to Timefold and I have this use case, combination of VRP with driver scheduling. I hope the community could provide some guidance and links to resources.

In the docs I came across "multi-stage planning" (I just thought my use case falls into this, but I might be wrong), but I can't seem to find anything further like examples as my starting point.

Thanks in advance.

**Comment by ge0ffrey** [comment]
If they're different types of vehicles, and drivers with different types of skills, then yes I would probably treat them as 2 planning problems and handle it as a multi-stage planning problem. Especially because the drivers will need to know weeks in advance about their shifts and the vehicle routes can be decided minutes or hours in advance.
